packages/block-library/src/latest-posts/index.php

```diff
@@ -158,7 +158,7 @@ function render_block_core_latest_posts( $attributes ) {
 
 			$list_items_markup .= sprintf(
 				'<div class="wp-block-latest-posts__post-full-content">%1$s</div>',
-				$post_content
+				esc_html( $post_content )
```

Comment:
I think the content shouldn't be escaped here as it may contain HTML. It's already sanitized by `wp_kses_post()` in line 163. The same probably applies to the excerpt.

Reply:
Thanks. Good spot.